scenarios/happypath/configbranch/configbranch_change.go

```diff
@@ -0,0 +1,164 @@
+package configbranch
+
+import (
+	"strings"
+
+	apiclient "github.com/equinor/radix-cicd-canary/generated-client/client/application"
+	models "github.com/equinor/radix-cicd-canary/generated-client/models"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/array"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/config"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/env"
+	envUtil "github.com/equinor/radix-cicd-canary/scenarios/utils/env"
+	httpUtils "github.com/equinor/radix-cicd-canary/scenarios/utils/http"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/job"
+	"github.com/equinor/radix-cicd-canary/scenarios/utils/test"
+	log "github.com/sirupsen/logrus"
+)
+
+var logger *log.Entry
+
+type expectedStep struct {
+	name       string
+	components []string
+}
+
+// Change Tests that radixconfig is read from the branch defined as configBanch
+func Change(env envUtil.Env, suiteName string) (bool, error) {
+	logger = log.WithFields(log.Fields{"Suite": suiteName})
+
+	// Trigger build via web hook
+	ok := httpUtils.TriggerWebhookPush(env, config.App4ConfigBranch, config.App4CommitID, config.App4SSHRepository, config.App4SharedSecret)
+	if !ok {
+		return false, nil
+	}
+	logger.Infof("First job was triggered")
+
+	// Get first job
+	ok, jobSummary := test.WaitForCheckFuncOrTimeout(env, func(env envUtil.Env) (bool, interface{}) {
+		return job.IsListedWithStatus(env, config.App4Name, "Running")
+	})
+
+	if !ok {
+		log.Errorf("Could not get listed job for application %s status \"%s\" - exiting.", config.App4Name, "Running")
+		return false, nil
+	}
+
+	jobName := (jobSummary.(*models.JobSummary)).Name
+	logger.Infof("First job name: %s", jobName)
+
+	ok, status := test.WaitForCheckFuncOrTimeout(env, func(env envUtil.Env) (bool, interface{}) {
+		return job.IsDone(env, config.App4Name, jobName)
+	})
+
+	if !ok {
+		return false, nil
+	}
+	if status.(string) != "Succeeded" {
+		return false, nil
+	}
+
+	logger.Info("First job was completed")
+
+	expectedSteps := []expectedStep{
+		{name: "clone-config", components: []string{}},
+		{name: "config-2-map", components: []string{}},
+		{name: "radix-pipeline", components: []string{}},
+		{name: "clone", components: []string{}},
+		{name: "build-www", components: []string{"www"}},
+		{name: "scan-www", components: []string{"www"}},
+	}
+
+	if ok := validateJobSteps(env, jobName, expectedSteps); !ok {
```

Comment:
Suggestion to return `error` from the method `validateJobSteps` and check here: if it is not `nil` - return with `false` and with this error.